ansible/playbooks/operator-pipeline-integration-tests.yml

```diff
@@ -101,6 +102,14 @@
               tags:
                 - test-release-pipeline
 
+        - name: Verify the Bundle was released to image indices
+          vars:
+            registry_access_token: "{{ stage_operator_pipelines_test_service_account_access_token }}"
+            ocp_versions_range: "v4.10"
```

Comment:
Why is this different from the range in the community test?

Reply:
There is a different version specified for community and isv test operators in the operator metadata annotations:
- `com.redhat.openshift.versions: v4.10` in `test-e2e-operator`
- ` com.redhat.openshift.versions: v4.11-v4.14` in `e2e-community-test-operator`